server/py/services/alerts/tests/unit/api/test_alerts.py

```diff
@@ -30,6 +30,69 @@ def test_store_alerts(self, db: Session, client: TestClient):
         project = "test-alerts"
         alert_name = "alert-name"
         self._create_project(db, project)
+        alert_config = self._create_alert_config(alert_name, project)
+        resp = client.put(
+            STORE_ALERTS_PATH.format(project=project, name=alert_name),
+            json=alert_config.dict(),
+        )
+        assert resp.status_code == HTTPStatus.OK.value
+
+        resp = client.get(
+            ALERTS_PATH.format(project=project),
+        )
+        assert resp.status_code == HTTPStatus.OK.value
+        alerts = resp.json()
+        assert len(alerts) == 1
+        assert alerts[0]["name"] == alert_name
+
+    def test_list_alerts_for_all_projects(self, db: Session, client: TestClient):
+        for i in range(2):
+            project = f"test-alerts-{i}"
+            alert_name = f"alert-name-{i}"
```

Comment:
Why not calling alert the same name for both project? It seems to be good edgecase to check

Reply:
Done 👍